From 284f635596eaf2bd557adf627a095e4ab2b2a8d6 Mon Sep 17 00:00:00 2001
From: Shilin Victor <radicaldreamer00001@gmail.com>
Date: Fri, 29 Mar 2019 22:51:38 +0300
Subject: [PATCH 19/45] 
 frameworks-av_WW010-Increase-shared-heap-size-to-8MB.patch

Change-Id: I78382d3757ff965be05cc75a54d7fa22f23f4c92
---
 services/audioflinger/AudioFlinger.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/audioflinger/AudioFlinger.h b/services/audioflinger/AudioFlinger.h
index 3e9a0885a..9241be1dc 100644
--- a/services/audioflinger/AudioFlinger.h
+++ b/services/audioflinger/AudioFlinger.h
@@ -92,7 +92,7 @@ static const nsecs_t kDefaultStandbyTimeInNsecs = seconds(3);
 // Max shared memory size for audio tracks and audio records per client process
 static const size_t kClientSharedHeapSizeBytes = 1024*1024;
 // Shared memory size multiplier for non low ram devices
-static const size_t kClientSharedHeapSizeMultiplier = 4;
+static const size_t kClientSharedHeapSizeMultiplier = 8;
 
 #define INCLUDING_FROM_AUDIOFLINGER_H
 
-- 
2.11.0

